lib/omniauth/strategies/open_id.rb

```diff
@@ -27,6 +27,7 @@ class OpenID
       option :store, ::OpenID::Store::Memory.new
       option :identifier, nil
       option :identifier_param, 'openid_url'
+      option :login_entrypoint
```

Comment:
Wonder if we should follow the omniauth-identity naming scheme and call this `on_login` (& default it to `nil` explicitly)